bindings/java/iota-client-java/src/main/java/org/iota/types/ClientConfig.java

```diff
@@ -5,14 +5,313 @@
 
 import com.google.gson.JsonObject;
 
-public class ClientConfig extends AbstractObject {
+public class ClientConfig {
 
-    public ClientConfig(JsonObject jsonObject) {
-        super(jsonObject);
+    private String primaryNode;
+    private String primaryPoWNode;
+    private String[] nodes;
+    private String[] permanodes;
+    private boolean nodeSyncEnabled = true;
+    private NodeSyncInterval nodeSyncInterval = new NodeSyncInterval().withSecs(60);
+    private boolean quorum;
+    private int minQuorumSize = 3;
+    private int quorumThreshold = 66;
+    private String network;
+    private String networkId;
+    private String bech32HRP = "rms";
+    private double minPoWScore = 4000.0;
+    private boolean localPow = true;
+    private boolean fallbackToLocalPow = true;
+    private int tipsInterval = 15;
+    private RentStructure rentStructure = new RentStructure(500, 10, 1);
+    private ApiTimeout apiTimeout = new ApiTimeout().withSecs(15);
+    private RemotePowTimeout remotePowTimeout = new RemotePowTimeout().withSecs(100);
```

Comment:
Why do we need the values here?

Reply:
But this will be set automatically during deserialization, so you don't have to do this on the Java side

Reply:
i think you're right, we don't need to set them there..